master/buildbot/test/fake/fakedb.py

```diff
@@ -2322,7 +2322,11 @@ def getMaster(self, masterid):
         return defer.succeed(None)
 
     def getMasters(self):
-        return defer.succeed(sorted(self.masters.values()))
+        def masterKey(master):
+            return master['id']
+
+        return defer.succeed(sorted(self.masters.values(),
+                                    key=masterKey))
```

Comment:
I disagree with you here.  I find lambdas hard to read.  In this case
the masterKey function is only two lines, and is right above this line.